pom.xml

```diff
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>nexus-plugins</artifactId>
-    <version>3.5.1-02</version>
+    <version>3.5.2-01</version>
```

Comment:
If you can, do these as a separate PR (for the pom update and for the README) and I'll get those merged in quicker. Thanks for keeping this up to date, ps!

Reply:
Ah that's strange that you received this update as I did not issue a merge request ...
Maybe something that I did not understand with github ...

Reply:
JSYK when you continue to commit in the branch with which you open a PR, the PR is updated. 
In this case you open this PR on you master branch, so every other commit you do on your master branch will be visible here and update this PR. 
Advice: when you want to work on a feature and open a PR, work on a new branch and open the pull request on that branch. That way you can continue to work on your master without updating the PR. 
Hopes it was clear and it helps. 
Thank you for this PR by the way !

Reply:
I see, I was supposing something like this, thanks a lot for your help !!